src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -186,13 +221,6 @@ public List<PackageManagerSearchElement> ListAll()
             return CachedPackageList;
         }
 
-        public List<PackageManagerSearchElement> Search(string search, int maxNumSearchResults)
-        {
-            return Model.Search(search, maxNumSearchResults)
-                               .Select((header) => new PackageManagerSearchElement(Model, header))
-                               .ToList();
-        }
-
```

Comment:
Remote package search.  This isn't in use.  Removing it here to prevent confusion.